src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const heads = document.querySelectorAll('th');
+const rows = document.querySelectorAll('tr');
+const arr = [];
+const obj = {};
+
+rows.forEach((row, index) => {
+  if (index !== 0 && index !== rows.length - 1) {
+    obj.name = row.children[0].innerText;
+    obj.position = row.children[1].innerText;
+    obj.age = row.children[2].innerText;
+    obj.salary = row.children[3].innerText;
+
+    arr.push(Object.assign({}, obj));
+  }
+});
+
+const sortTable = (sortedBy) => {
+  if (sortedBy === 'names') {
+    arr.sort((a, b) => a.name.localeCompare(b.name));
+  } else if (sortedBy === 'positions') {
+    arr.sort((a, b) => a.position.localeCompare(b.position));
+  } else if (sortedBy === 'ages') {
+    arr.sort((a, b) => a.age - b.age);
+  } else if (sortedBy === 'salaries') {
+    arr.sort((a, b) => parseFloat(a.salary.slice(1))
+    - parseFloat(b.salary.slice(1)));
+  }
+
+  rows.forEach((row, index) => {
```

Comment:
try to use `map` method here

Reply:
> try to use `map` method here

I'm sorry, I didn't understand this task(